### ISSUE TITLE
Double Cuffing dealing DoT

### ISSUE THREAD

**Issue by auburncivilnetworks** [civilnetworks-projects/SCP_Public_Development#534]
### Description
When Cuffing someone with Elastic Restraints and Metal Handcuffs at the same time, dragging said individual around will deal damage depending on their velocity.

### How to Replicate
- Elastic Cuff Someone
- Metal Handcuff that person
- Drag them around

**Comment by CNCanoon** [comment]
_To be checked by Content._

**Comment by CNCanoon** [comment]
Confirmed to still be an issue.

**Comment by McYeke** [comment]
I dont think this is still an issue, have been dragged double cuffed a lot and taken nil dmg
